sdk/src/main/java/io/dapr/exceptions/DaprError.java

```diff
@@ -20,12 +24,20 @@
    */
   private String message;
 
+  /**
+   * Error code from gRPC.
+   */
+  private Integer code;
```

Comment:
This is needed because the resulting JSON is different for a gRPC service.

Reply:
How is this field set? 
I think this is set on json parsing?

Reply:
It is set on JSON parsing. I found out about this in ITs.